pulpcore/pulpcore/app/viewsets/content.py

```diff
@@ -10,34 +10,54 @@
 from pulpcore.app.viewsets import NamedModelViewSet
 
 
-class ContentFilter(filterset.FilterSet):
```

Comment:
Ignore this, github is confused.  I just moved the filters around to sit next to their viewsets, and changed the docstrings to be consistent with the changes Austin wanted in the filter overhaul PR